*error_aggregateTwoErrors.js:*
throw aggregateTwoErrors(err, originalError);
^

AggregateError: original
    at Object.<anonymous> (*error_aggregateTwoErrors.js:*:*) {
  code: 'ERR0',
  [errors]: [
    Error: original
        at Object.<anonymous> (*error_aggregateTwoErrors.js:*:*) {
      code: 'ERR0'
    },
    Error: second error
        at Object.<anonymous> (*error_aggregateTwoErrors.js:*:*) {
      code: 'ERR1'
    }
  ]
}

Node.js *
